src/garage/experiment/local_tf_runner.py

```diff
@@ -319,6 +323,39 @@ def train(self,
             pause_for_plot=pause_for_plot,
             start_epoch=0)
 
+        return self._train()
+
+    def _step_epochs(self):
```

Comment:
This should be a public API, so call the method `step_epochs()` (without the underscore).
The most important thing to describe in this docstring is the surprising behaviour. In particular, the fact that this method uses the `step_epoch_args` (or after renaming, the `itr` and `path` fields), at the end of each epoch.

The docstring should also describe that it yields each epoch.